frontend_tests/puppeteer_tests/02-message-basics.ts

```diff
@@ -351,9 +349,9 @@ async function test_stream_search_filters_stream_list(page: Page): Promise<void>
     await test_search_venice(page);
 
     // Search for brginning of "Verona".
-    await page.evaluate(() =>
-        $(".stream-list-filter").expectOne().trigger("focus").val("ver").trigger("input"),
-    );
+    await page.$eval(".stream-list-filter", (el) => {
+        (el as HTMLInputElement).value = "ver";
+    });
```

Comment:
@Riken-Shah I think this one has a semantics change -- we're no longer focusing and typing in the box; I could imagine that affecting the behavior of the production code.  Though maybe we should just replace this with the standard Puppeteer functions for typing these characters into an input box, which will likely be better than either version here.